internal/databases/mongo/models/backup.go

```diff
@@ -33,6 +38,83 @@ func (b *Backup) IsPermanent() bool {
 	return b.Permanent
 }
 
+func (b *Backup) PrintableFields() []printlist.TableField {
+	prettyStartTime := internal.PrettyFormatTime(b.StartLocalTime)
+	prettyFinishTime := internal.PrettyFormatTime(b.FinishLocalTime)
+	return []printlist.TableField{
+		{
+			Name:       "name",
+			PrettyName: "Name",
+			Value:      b.BackupName,
+		},
+		{
+			Name:       "type",
+			PrettyName: "Type",
+			Value:      b.BackupType,
+		},
+		{
+			Name:       "version",
+			PrettyName: "Version",
+			Value:      b.MongoMeta.Version,
+		},
+		{
+			Name:        "start_time",
+			PrettyName:  "Start time",
+			Value:       internal.FormatTime(b.StartLocalTime),
+			PrettyValue: &prettyStartTime,
+		},
+		{
+			Name:        "finish_time",
+			PrettyName:  "Finish time",
+			Value:       internal.FormatTime(b.FinishLocalTime),
+			PrettyValue: &prettyFinishTime,
+		},
+		{
+			Name:       "hostname",
+			PrettyName: "Hostname",
+			Value:      b.Hostname,
+		},
+		{
+			Name:       "start_ts",
+			PrettyName: "Start Ts",
+			Value:      b.MongoMeta.Before.LastMajTS.String(),
```

Comment:
`b.MongoMeta.Before.LastMajTS.ToBsonTS()`

Reply:
I was just confused how it should be printed if `bson/primitive/Timestamp` stuct doesn't implement `String()`. Just with the standard Go "%v" struct formatter?

Reply:
Did it